static/app/views/projectInstall/platform.spec.tsx

```diff
@@ -136,5 +138,46 @@ describe('ProjectInstallPlatform', function () {
         name: 'Configure Browser JavaScript SDK',
       })
     ).toBeInTheDocument();
+
+    expect(await screen.findByText('Take me to Issues')).toBeInTheDocument();
```

Comment:
Since we expect them all to appear, can we not use getByText for all 3?

Reply:
ah yes, we can. changed!